tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,710 @@
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test Filters Package System tests:
+        foglamp-south-http-south south plugin
+        foglamp-filter-expression, foglamp-filter-python35, foglamp-filter-simple-python, foglamp-filter-ema filter plugins
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import subprocess
+import http.client
+import json
+import pytest
+import os
+import time
+import urllib.parse
+from pathlib import Path
+
+PROJECT_ROOT = ""
+SCRIPTS_DIR_ROOT = ""
+ASSET_NAME_PY35 = "end_to_end_py35"
+ASSET_NAME_SP = "end_to_end_sp"
+ASSET_NAME_EMA = "end_to_end_ema"
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 20
```

Comment:
Most of system tests are covered with same input value. Could you please test with different value?